Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -815,6 +844,12 @@ private void SkipToNextObject()
                     var inputManager = (KeysInputManager)Ruleset.InputManager;
                     inputManager.ReplayInputManager.HandleSkip();
                 }
+
+                // Stop all playing sound effects and move NextSoundEffectIndex ahead.
+                CustomAudioSampleCache.StopAll();
+                while (NextSoundEffectIndex < Map.SoundEffects.Count &&
```

Comment:
Code is duplicated here. As mentioned, you can make this a method and call it here as well.